### ISSUE TITLE
Contract methods interactions

### ISSUE THREAD

**Issue by vladislavbogomolov** [iexbase/tron-api#65]
It's possible to interact with Contracts? https://developers.tron.network/reference#contract

**Comment by atlaschiew** [comment]
yes it is very simple with this library , that's a triggersmartcontract() in tron-api-master\src\TransactionBuilder.php

or you may refer to https://www.btcschools.net/tron/tron_create_trc20_tx.php about how do i implement it

Atlas Chiew
btcschools.net

**Comment by chuyijia** [comment]
> yes it is very simple with this library , that's a triggersmartcontract() in tron-api-master\src\TransactionBuilder.php
> 
> or you may refer to https://www.btcschools.net/tron/tron_create_trc20_tx.php about how do i implement it
> 
> Atlas Chiew btcschools.net

That's a very nice and useful site!but I'm still confused about how to generate params under abi rules when interacting with the tron contract, would u mind wirting the sample code for me  using   transfer function.

**Comment by atlaschiew** [comment]
> > yes it is very simple with this library , that's a triggersmartcontract() in tron-api-master\src\TransactionBuilder.php
> > or you may refer to https://www.btcschools.net/tron/tron_create_trc20_tx.php about how do i implement it
> > Atlas Chiew btcschools.net
> 
> That's a very nice and useful site!but I'm still confused about how to generate params under abi rules when interacting with the tron contract, would u mind wirting the sample code for me using transfer function.

hi, u may drop me a message as issue in my repo :p
